file-reader/src/main/java/com/bobocode/FileReaders.java

```diff
@@ -12,6 +23,27 @@
      * @return string that holds whole file content
      */
     public static String readWholeFile(String fileName) {
-        throw new UnsupportedOperationException("It's your job to make it work!"); //todo
+        Path filePath = createPathFromFileName(fileName);
+        try (Stream<String> fileLinesStream = openFileLinesStream(filePath)) {
+            return fileLinesStream.collect(joining("\n"));
+        }
+    }
+
+    private static Stream<String> openFileLinesStream(Path filePath) {
+        try {
+            return Files.lines(filePath);
+        } catch (IOException e) {
+            throw new FileReaderException("Cannot create stream of file lines!", e);
+        }
+    }
+
+    private static Path createPathFromFileName(String fileName) {
+        Objects.requireNonNull(fileName);
+        URL fileUrl = FileReaders.class.getClassLoader().getResource(fileName);
```

Comment:
"getResource()"
Does this method retrieve "resources" folder?

Reply:
**Not only**, in our case files from folder `resources` will be in the classpath, and that method allows to get a `URL` object of any kind of resource (any file) by its name. 

You can check its javadoc for more details.